### ISSUE TITLE
Gets into the NaN loss value after some epochs!!

### ISSUE THREAD

**Issue by imtiazziko** [wasidennis/AdaptSegNet#54]
Hello @wasidennis

1. I  get into the **NaN** loss value while the training is going on from epoch 12396. There can be the instability of the adversarial or learning rate or else but not sure which one of it. I used the same setting for single level DeepLab model as per your code. What can be the problem here? 

![issue_adaptseg](https://user-images.githubusercontent.com/20689792/55973610-ef662300-5c53-11e9-90f1-875e0e9e1f32.png)

2.  Also how did you decide on the **early stopping** epoch number 149999?

3. I really can not reproduce the result of 41.4 % reported in the paper (GTA5-Cityscapes). Can you help me on that?

Thank you very much. Looking forward to your answers.

**Comment by wasidennis** [comment]
We have not experienced any issue like your NaN case. It seems that our code has been modified based on your screenshot. Could you double check whether there are any differences from our original code or provide more information about this issue? Thanks!

**Comment by imtiazziko** [comment]
Thanks for replying. But I did not change your code.  I think you might be able to see this issue iby printing the logs like I did.

**Comment by wasidennis** [comment]
@imtiazziko In the original implementation, we are also printing all the losses, but we never have this issue before in any experiments. It also looks like the segmentation loss is really hight before going to NaN, which is not normal. Could you elaborate more details, e.g., training hyper-parameters, which pre-trained weights are used?
